circle.yml

```diff
@@ -1032,6 +1033,47 @@ jobs:
           working_directory: test-binary
           command: $(npm bin)/cypress info
 
+  test-types-cypress-and-jest:
+    parameters:
+      executor:
+        description: Executor name to use
+        type: executor
+        default: cy-doc
```

Comment:
I had to add it because our default YML `<<*defaults` inclusion block already has `parameters` object, thus it was complaining about `executor`, so I had to add it here to avoid the Circle validation